tables/es-translation.cti

```diff
@@ -42,3 +42,142 @@ capsletter 46
 begcapsword 46-46
 
 # ----------------------------------------------------------------------------------------------
+
+# Emphasis
+emphclass italic
+emphclass underline
+emphclass bold
+emphclass script
+
+begemph bold 35
+endemph bold 35
+begemph italic 35-35
+endemph italic 35-35
+begemph underline 56-35
+endemph underline 56-35
+begemph script 56-35
+endemph script 56-35
+
+# Multiple character definitions
+
+always        \x002F\x002F  6-26-2
+always        \x005C\x005C  5-35-3
+always        \x007C\x007C  456-123
+
+# Numbers followed by letters a-j should be prefixed with dot5
```

Comment:
it would be nice to have a yaml example or at least some context to understand where this could occur, at least in literary Braille all numbers are separated from text by whitespace, prefixed by the number sign.

Reply:
This is something that most tables implement with `numericnocontchars abcdefghij`. This can occur in any number that contains a letter, e.g. a house number "37b".